### PR TITLE
Remove OSTYPE check and update README file

### PR DESCRIPTION
**Pull request by cimurah**
Remove OSTYPE check in Dockerfile since it is not
necessary. Its value is empty when image is being built.

Cleaned up commands in README file.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ EXPOSE 8080
 # work around https://github.com/docker-library/openjdk/blob/0584b2804ed12dca7c5e264b5fc55fc07a3ac148/8-jre/slim/Dockerfile#L51-L54
 RUN mkdir -p /usr/share/man/man1
 
-RUN if [[ "$OSTYPE" == "linux-gnu"* ]]; then addgroup --gid $GROUP_ID docker; fi
+RUN addgroup --gid $GROUP_ID docker
```
Comment (samwilson) [inline docker/Dockerfile:13]: I don't have Windows so can't test this, but I assume it's all good!

File: README.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Installation
 
   * Establish an SSH tunnel to the replicas (only necessary on local environments)
 ```bash
-$ ./php bin/console toolforge:ssh
+$ php bin/console toolforge:ssh
```
Comment (samwilson) [inline README.md:51]: This should probably match the below and be just `./bin/console …`

**Review by samwilson — APPROVED** [review]
Looks good to me.

File: README.md (hunk 2 above)
Comment (cimurah) [reply README.md]: Done. Thank you!

**Pull request merged**